common/internal_model/src/main/scala/weco/catalogue/internal_model/index/ImagesIndexConfig.scala

```diff
@@ -19,126 +16,22 @@ object ImagesIndexConfig extends IndexConfigFields {
-          // production events
-          labelField("production.label"),
-          dateField("production.dates.range.from"),
-          // relations
-          canonicalIdField("partOf.id"),
-          multilingualFieldWithKeyword("partOf.title"),
-          // availabilities
-          keywordField("availabilities.id"),
-          // collection path
-          textField("collectionPath.path")
-            .analyzer(exactPathAnalyzer.name)
-            .fields(
-              keywordField("keyword"),
-              textField("clean").analyzer(cleanPathAnalyzer.name)
-            ),
-          textField("collectionPath.label")
-            .analyzer(asciifoldingAnalyzer.name)
-            .fields(
-              keywordField("keyword"),
-              lowercaseKeyword("lowercaseKeyword"),
-              textField("cleanPath").analyzer(cleanPathAnalyzer.name),
-              textField("path").analyzer(exactPathAnalyzer.name)
-            ),
-          // reference number
-          keywordField("referenceNumber"),
-        )
-
-      val query = objectField("query")
-        .fields(
-          canonicalIdField("id"),
-          sourceIdentifierField("sourceIdentifier.value"),
-          keywordField("locations.license.id"),
-          inferredData,
-          sourceWork
-        )
-
-      // This field contains the display documents used by aggregations.
-      //
-      // The values are JSON documents that can be included in an AggregationBucket from
-      // the API, but those documents are then encoded as JSON strings so they can be
-      // returned as single values from an Elasticsearch terms aggregation.
-      //
-      // See https://github.com/wellcomecollection/docs/tree/main/rfcs/049-catalogue-api-aggregations-modelling
-      val aggregatableValues = objectField("aggregatableValues")
-        .fields(
-          keywordField("locations.license"),
-          keywordField("source.contributors.agent.label"),
-          keywordField("source.genres.label"),
-          keywordField("source.subjects.label")
-        )
-
-      val fields = Seq(
-        dateField("modifiedTime"),
-        display,
-        query,
-        aggregatableValues
-      )
-
       // Here we set dynamic strict to be sure the object vaguely looks like an
       // image and contains the core fields, adding DynamicMapping.False in places
       // where we do not need to map every field and can save CPU.
       val mapping = {
-        val version = BuildInfo.version.split("\\.").toList
-        properties(fields)
-          .dynamic(DynamicMapping.Strict)
-          .meta(Map(s"model.versions.${version.head}" -> version.tail.head))
+        MappingDefinition(
+          rawSource = Some(
+            IndexMapping(
+              propertiesJson = Source
+                .fromInputStream(
+                  getClass.getResourceAsStream("/imagesIndexProperties.json")
+                )
+                .mkString,
```

Comment:
Minor: in scala-libs we have a LocalResources trait which wraps this operation in a function. You can't access it here, but maybe it would be clearer to create a function for reading a resource as a string?